Monika After Story/game/script-topics.rpy

```diff
@@ -1570,13 +1573,53 @@ label monika_yuri:
     m 2euc "But if you do happen to be into the yandere type..."
     show monika 5eua at t11 zorder MAS_MONIKA_Z with dissolve
     m 5eua "I can try acting a little more creepy for you. Ehehe~"
-    m "Then again..."
-    show monika 4hua at t11 zorder MAS_MONIKA_Z with dissolve
-    m 4hua "There's already nowhere else for you to go, or anyone for me to get jealous over."
-    m 2etc "Is this a yandere girl's dream?"
-    if not persistent._mas_pm_cares_about_dokis:
-        m 1eua "I'd ask Yuri if I could."
-    return
+    $ _history_list.pop()
+    menu:
+        m "I can try acting a little more creepy for you. Ehehe~"
+        "Actually... I'm into yanderes":
+            $ persistent._mas_pm_likes_yanderes = 1
+            m 1sub "Now that I'm hearing that I'm quite surprised you prefer me over Yuri"
```

Comment:
Missing a period. 

Add a comma after "that".

Normally, Monika has no doubt in the player's love for her, so I don't know if this line fits.

Reply:
I don't think this line introduces doubt... Moni says that she's surprised, that the player prefers her over Yuri, even if the player is into yanderes...